greenplumpython/table.py

```diff
@@ -11,10 +11,12 @@ def __init__(
         parents: Iterable["Table"] = [],
         name: Optional[str] = None,
         db: Optional[db.Database] = None,
+        order_by: Optional[Union[Iterable, dict]] = None,
```

Comment:
when handle `group by`, it was put into the query string and created a new table

https://github.com/greenplum-db/GreenplumPython/blob/cf0ce562390836f17b656645770e83156b78ae16/greenplumpython/func.py#L51

```
        orig_func_table = Table(
            " ".join(
                [
                    f"SELECT {str(self)}",
                    "," + group_by_columns if group_by_columns != "" else "",
                    from_caluse,
                    group_by_clause,
                ]
            ),
            db=self._db,
            parents=parents,
        )
```

Does it make sense to treat them differently? Is there any concern to put the `order by` clause in the `query` str directly?